### PR TITLE
Fix script issues

### PR DESCRIPTION
**Pull request by KB5201314**

### CODE REVIEW

**Review by oxr463 — APPROVED** [review]

**Review by oxr463 — COMMENTED** [review]
How can we get shellcheck to test the hook? Also, we should add it to the CI/CD workflow.

**Comment by KB5201314** [comment]
> How can we get shellcheck to test the hook? Also, we should add it to the CI/CD workflow.

For this hook let's run shellcheck it manually, since it doesn't matchs "*.sh“. 😂

I'll add shellcheck to the CI later.

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -20,10 +20,30 @@ jobs:
...
+          else
+            before_commit=${{ github.event.before }}
+          fi
+        fi
+        after_commit=$GITHUB_SHA
+        echo "before_commit: ${before_commit} after_commit: ${after_commit}"
+        git fetch origin "${before_commit}" --depth=1
+
+        # Run Shell code issues check
+        git diff --name-only "${before_commit}".."${after_commit}" | grep '\.rs$' | xargs --no-run-if-empty ls -1df 2>/dev/null | xargs --no-run-if-empty shellcheck || { echo "shell check failed and exit"; exit 1; }
```
Comment (oxr463) [inline .github/workflows/rust.yml:40]:
```suggestion
        git diff --name-only "${before_commit}".."${after_commit}" | grep '\.sh$' | xargs --no-run-if-empty ls -1df 2>/dev/null | xargs --no-run-if-empty shellcheck || { echo "shell check failed and exit"; exit 1; }
```

**Review by oxr463 — CHANGES_REQUESTED** [review]
I think this needs to be changed for shell scripts.

**Pull request merged**